docker/images/build.sh

```diff
@@ -1,14 +1,18 @@
 #!/usr/bin/env bash
 set -x
 
+IMAGE=${1:-backend}
+
+OUTPUT="type=registry"
+
+if [ "--local" = "$2" ]; then
+    OUTPUT="type=docker"
+fi
+
 DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
This option (not sure if it's used in fact), but it seems to be deprecated; I suggest to remove it.
<img width="921" height="249" alt="image" src="https://github.com/user-attachments/assets/c799eedd-f025-4420-8daa-29463fd1d501" />